docker/README.md

```diff
@@ -51,8 +51,43 @@ $ docker/build
 ```sh
 $ docker/server
 ```
-* The default admin user is 'ofn@example.com' with the password 'ofn123'.
-* View the app in the browser at `http://localhost:3000`.
+* Open a new terminal and access the Rails console in the container:
+```sh
+$ sudo docker-compose exec web bundle exec rails console
+```
+* In the Rails console, create a new user account:
+```sh
+$ > user = Spree::user.new(email: 'ofn@gmail.com', password: 'ofn123')
```

Comment:
This shouldn't be necessary. The seed data for the database contains a user already. Did that fail in your case?

Reply:
Yes!!, The seeded data did not work in my case. I gues there is no need  for this then if it works for others

Reply:
Yes, but it would be good to know why the seed data didn't work. Maybe there's another bug hiding in the setup.